pps/pps+private.go

```diff
@@ -13,11 +10,11 @@ import (
 	"github.com/CRAB-LAB-NTNU/PPS-BS/types"
 )
 
-func (pps *PPS) setIdealAndNadir(generation int) {
+func (pps *PPS) setIdealAndNadir() {
 	ip, np := biooperators.CalculateNadirAndIdealPoints(pps.moea.Population())
-
-	pps.idealPoints[generation] = ip
-	pps.nadirPoints[generation] = np
+	//Append in stead of initialising arrays at a fixed size to
+	pps.idealPoints = append(pps.idealPoints, ip)
+	pps.nadirPoints = append(pps.nadirPoints, np)
```

Comment:
Had issues setting a static array size for different population sizes and maxFuncEvals, changed this to append each generation in stead.